tests/libs/util/wer_report.h

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Does every test app link to this library?

Reply:
This is included in catch_wrapper.hpp, which is used by every test.